code_map.py

```diff
@@ -181,7 +189,12 @@ def reset_layout(reduce):
     width = 1 - settings().get("codemap_width")
 
     map_view = get_code_map_view()
-    if map_view:
+    try:
+        alone_in_group = len(w.views_in_group(get_group(map_view))) == 1
+    except:
+        alone_in_group = True
+
```

Comment:
I added a try block because I had an error in an occasion. map_view had been closed and was not there. I think this could need some rework, maybe just check if the last group has no more views.